healthcare/ich_segmentation/cassandra/cassandra-upstream-chart/files/conf/jmxremote.password

```diff
@@ -0,0 +1 @@
+jmx_user cassandra
```

Comment:
is this a  hardcoded password? if so please remove it

Reply:
Those are example files, I added .example extension on those files and I changed the passwor to something like "<PASSWORD_FOR_DB>".